app/controllers/concerns/attachments_copier.rb

```diff
@@ -7,14 +7,16 @@ def copy_attachments(record)
 
       if attachment
         new_attachment = attachment.copy_to(record.node)
-        new_filename = new_attachment.url_encoded_filename
-        new_path = full_screenshot_path.gsub(
-          /nodes\/[0-9]+\/attachments\/.+/,
-          "nodes/#{new_attachment.node_id}/attachments/#{new_filename}"
-        )
-
-        record.content = record.content.gsub(full_screenshot_path, new_path)
+        record.content = updated_record_content(record.content, full_screenshot_path, new_attachment)
       end
     end
   end
+
+  def updated_record_content(record_content, full_screenshot_path, new_attachment)
```

Comment:
`updated_record_content` doesn't quite describe what this method does. What about `update_attachment_link` or `update_attachment_reference`, or something else?